tests/test_runs/test_run_functions.py

```diff
@@ -367,7 +367,7 @@ def _check_sample_evaluations(
                                 # and/or measurements are not as accurate.
                                 # Either way, windows seems to get an eval-time
                                 # of 0 sometimes.
-                                self.assertGreater(evaluation, 0)
+                                self.assertGreaterEqual(evaluation, 0)
```

Comment:
What is the scenario where this is needed now?
If we weaken the test like this, the conditional on checking the platform is also no longer necessary.
But this also makes the test mostly pointless as it could just not record any time in all scenarios and pass.

Reply:
I admit I wasn't entirely sure about this and can revert it

Reply:
But I imagine you must have ran into this?

Reply:
well, there were some unit tests stochastically failing with the assertion error that `0.0 is not greater than 0`
while passing for other runs
I was checking if loosening the strictness of this check fixes it entirely, but as you point out, that makes the point of the check moot 
so I reverted it